roles/etcd/defaults/main.yml

```diff
@@ -10,4 +10,10 @@ etcd_address: "{{ ansible_default_ipv4.address }}"
 etcd_initial_cluster: "https://localhost:2380"
 etcd_initial_cluster_state: "new"
 etcd_client_address: 127.0.0.1
-tls_cipher_suites: ""
\ No newline at end of file
+etcd_certs:
+  - etcd/ca.crt
+  - apiserver-etcd-client.crt
+  - apiserver-etcd-client.key
+etcd_on_control_plane: true
+tls_cipher_suites: ""
+upgrade: false
```

Comment:
ansible-lint recommends setting this variable name as `etcd_upgrade`. I think is a good idea.